src/core/graphQL-interface/GraphQL.ts

```diff
@@ -509,36 +506,10 @@ function createResolvers(core: PerspectivismCore) {
         },
 
         LanguageHandle: {
-            //@ts-ignore
-            constructorIcon: async (language) => {
-                const code = await core.languageController.getConstructorIcon(language);
-                if (code) {
-                    return { code }
-                } else {
-                    return { code: "" }
-                }
-            },
-            //@ts-ignore
-            icon: async (language) => {
-                const code = await core.languageController.getIcon(language);
-                if (code) {
-                    return { code }
-                } else {
-                    return { code: "" }
-                }
-            },
             //@ts-ignore
             settings: async (language) => {
-                return JSON.stringify(core.languageController.getSettings(language))
+                return JSON.stringify(core.languageController.getSettings(language.address))
             },
-            //@ts-ignore
-            settingsIcon: async (language) => {
-                const code = await core.languageController.getSettingsIcon(language)
-                if(code)
-                    return { code }
-                else
-                    return null
-            }
```

Comment:
Why remove all of this?

Reply:
because whenever a LanguageHandle type was being returned, the Language object already had the Icons on it already so this was just making duplicate calls